docs/operating-scylla/nodetool-commands/cleanup.rst

```diff
@@ -20,6 +20,12 @@ To clean up the data of a specific node and specific keyspace, use this command:
 
    nodetool -h <host name> cleanup <keyspace>
 
+Options
+-------
+
+* ``--global`` or ``-g`` - Run cleanup on all nodes that requires it in the cluster
+
```

Comment:
We also have a cluster subcommand, could be exposed as `nodetool cluster cleanup`

Reply:
I moved to use  `nodetool cluster cleanup` instead.